evap/staff/tests/test_views.py

```diff
@@ -139,6 +140,21 @@ def test_questionnaire_edit(self):
         form.submit()
         self.assertTrue(UserProfile.objects.filter(username='lfo9e7bmxp1xi').exists())
 
+    def test_remove_course(self):
```

Comment:
Maybe rename the test to "test_reward_points_granting_message"?
Maybe let mommy set the username and compare to the (random) name in the assertion?